README.md

```diff
@@ -44,9 +44,14 @@ As a requirement, please make sure to:
       microk8s enable registry
       # ensure kubectl is configured to use microk8s
       microk8s.kubectl config view --raw > $HOME/.kube/config
+      snap alias microk8s.kubectl kubectl
 
 
-* ensure [`skaffold`](https://github.com/GoogleContainerTools/skaffold), [`container-structure-test`](https://github.com/GoogleContainerTools/container-structure-test) and [`docker`](https://docs.docker.com/engine/install/ubuntu/) are installed according to their documentation
+* ensure [`skaffold`](https://github.com/GoogleContainerTools/skaffold), [`container-structure-test`](https://github.com/GoogleContainerTools/container-structure-test), [`docker`](https://docs.docker.com/engine/install/ubuntu/) and [helm](https://helm.sh/docs/intro/install) are installed according to their documentation
+
+* initialise LXD
+
+      sudo lxd init --auto
```

Comment:
not sure i understand why lxd is part of this, but if it's for troubleshooting when u have docker and lxd running together we'd better have a separate section

Reply:
LXD is used by rockcraft and without a default profile you get an error when running `make dev`:

```
Command standard error output: b'Error: Failed instance creation
: Failed creating instance record: Failed initialising instance: Failed getting root disk:
 No root device could be found\n'
```

Reply:
aaah my bad, that is needed when we build the image

fair point, apologies, do u mind adding something along these lines?

```suggestion
* initialise LXD to use `rockcraft` during `skaffold build`

      sudo lxd init --auto
```

Reply:
Done, I've also added a comment to explain why we need to alias kubectl.